style.css

```diff
@@ -75,3 +87,309 @@ ul {
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-left: 34.5px;
+  margin-right: 34.5px;
+  grid-row: 2 / 3;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.work-item02 {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-left: 34.5px;
+  margin-right: 34.5px;
+  grid-row: 3 / 4;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.work-item03 {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-left: 34.5px;
+  margin-right: 34.5px;
+  grid-row: 4 / 5;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.work-item04 {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-left: 34.5px;
+  margin-right: 34.5px;
+  grid-row: 5 / 6;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.work-item05 {
+  width: 327px;
+  height: 474px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  margin-left: 34.5px;
+  margin-right: 34.5px;
+  grid-row: 6 / 7;
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+}
+
+.work-content {
+  width: 295px;
+  padding: 16px;
+  border-radius: 8px;
+  background-color: #fff;
+  display: grid;
+  gap: 16px;
+  grid-template-rows: 57px 32px 69px;
+  align-content: center;
+}
+
+.work-title {
+  width: auto;
+  margin: 0;
+  font-weight: 600;
+  font-size: 24px;
+  color: #3a4a42;
+  text-align: center;
+  grid-row: 1 / 2;
+}
+
+.cards {
+  width: 305px;
+  height: 32px;
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+  grid-row: 2 / 3;
+}
+
+.cards li {
+  background-color: #ebf0ee;
+  border-radius: 4px;
+  font-weight: 600;
+  font-size: 12px;
+  color: #3a4a42;
+  padding: 8px, 12px, 8px, 12px;
+}
+
+.see-project {
+  width: 98px;
+  height: 24px;
+  border-radius: 4px;
+  padding: 12px;
+  background-color: #36b37f;
+  font-weight: 500;
+  font-size: 17px;
+  color: #fff;
+  margin-left: 86.5px;
+  margin-right: 86.5px;
+  grid-row: 3 / 4;
+}
+
+.aboutme {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/* To add the background images */
+.bg-1 {
+  height: 127px;
+  width: 396px;
+  background-image: url("./Images/aboutme_frame02.png");
+  background-repeat: no-repeat;
+  background-position: 100% 0%;
+  display: flex;
+  justify-content: center;
+}
+
+.aboutme-heading {
+  margin: 0;
+  margin-top: 60px;
+  width: 370px;
+}
+
+.bg-2 {
+  height: 65px;
+  width: 396px;
+  background-image: url("./Images/aboutme_frame01.png");
+  background-repeat: no-repeat;
+  background-position: 0% 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.resume-button {
+  /* width: 135px; */
+  height: 48px;
```

Comment:
- [ ] As I pointed out above, in the previous comment, kindly make sure to remove every contented code lines to make you code clean and more professional.